addon/-private/adapters/rest-adapter.js

```diff
@@ -3,18 +3,18 @@
 */
 
 import Ember from 'ember';
-import Adapter from "ember-data/system/adapter";
+import Adapter from "ember-data/-private/system/adapter";
```

Comment:
Another way of putting this: to date Ember Data's public API has been strictly the globals output (which seems unchanged by this PR). This PR helps ensure that we are not inadvertently making many more of our internals "public" because the modules are importable and not marked as private.